src/user/integrations/jitsi.scss

```diff
@@ -52,34 +52,37 @@
     color:#000000;
     margin-bottom: 3px;
     margin-top: 5px;
-   // margin-left: 3px;
+    margin-left: 2px;
 }
 .form-input{
     font-family: Inter;
     font-style: normal;
     font-weight: 300;
     font-size: 13px;
     line-height: 16px;
-    //margin-left: 3px;
+    margin-left: 3px;
 }
 .form-content{
     margin-top:-2%;
     margin-bottom: 20px;
     justify-content:space-between;
 }
-.form-footer{
+.form-footer-jitsi{
```

Comment:
Remove unnecessary CSS classes that are not used, please.

Reply:
Removed unnecesarry css classes from jitsi.scss.